pkg/build/builder/sti_test.go

```diff
@@ -57,7 +58,7 @@ func newTestS2IBuilder(config testS2IBuilderConfig) *S2IBuilder {
 		&FakeDocker{
 			errPushImage: config.errPushImage,
 		},
-		"unix:///var/run/docker.sock",
+		"unix:///var/run/docker2.sock",
```

Comment:
?

Reply:
+1

Reply:
To ensure this doesn't go out to the docker socket if someone accidentally exposes a path